charts/logzio-telemetry/values.yaml

```diff
@@ -297,13 +297,12 @@ metricsConfig:
     prometheus:
       config:
         global:
-          scrape_interval: 15s
-          scrape_timeout: 10s
-          evaluation_interval: 10s
+          scrape_interval: 30s
```

Comment:
Maybe we should increase the timeout here to 60s? This is the standalone collector config